homeassistant/components/nsw_fuel_station/__init__.py

```diff
@@ -18,6 +19,8 @@
 DOMAIN = "nsw_fuel_station"
 SCAN_INTERVAL = datetime.timedelta(hours=1)
 
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
```

Comment:
Hmm, this is a platform type of integration, should this be using the no config schema?